parameter_set/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+cmake_minimum_required(VERSION 3.5)
+project(parameter_set)
+
+# Default to C++17
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+
+# Compile options set for the targets
+set(PRIVATE_COMPILE_OPTIONS -Wall -Wextra -Wpedantic)
+
+# find dependencies
+find_package(absl_status REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_ros REQUIRED)
+find_package(rcl REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+
+set(THIS_PACKAGE_INCLUDE_DEPENDS
+  rclcpp
+  sensor_msgs
+  absl_status
+)
+
+add_library(${PROJECT_NAME}
```

Comment:
Should be a shared library .?

Reply:
No, I think this will generally be statically linked.